indra/db/database_manager.py

```diff
@@ -277,7 +277,7 @@ class DBInfo(self.Base):
         class RawStatements(self.Base):
             __tablename__ = 'raw_statements'
             id = Column(Integer, primary_key=True)
-            uuid = Column(String(40))
+            uuid = Column(String(40), unique=True, nullable=False)
```

Comment:
This used to be here, but go lost in the churn (uuid went from just another column, to a primary key, and now back.)